scripts/watch.js

```diff
@@ -78,6 +86,16 @@ async function addOmniFiles(relPaths) {
 	};
 }
 
+async function notifyIPC() {
+	const pipesDir = path.normalize("/Users/doppler/Zotero default-beta/pipes/");
```

Comment:
Hard-coded path. Seems like we'd need to prompt for the data directory in order to keep this, but @adomasven and I were talking about removing the default IPC pipe anyway, since its only remaining use, Firefox connector switching, is long since obsolete.

What about using the HTTP server (server.js) instead? Then it could just assume the first-started instance on 23119 is the one to use (or we could take a different port via an environment variable, I guess). See server_connector.js for examples of how to create an endpoint by adding an object within `Zotero.Server.Endpoints`. (It can be defined in another file.)

Reply:
Oops, good spot - didn't mean to leave this in the PR, thanks! I'll experiment with HTTP approach.